pyogrio/tests/test_raw_io.py

```diff
@@ -394,7 +410,7 @@ def test_write_supported(tmpdir, naturalearth_lowres, driver):
     # we take the first record only.
     meta["geometry_type"] = "MultiPolygon"
 
-    filename = tmpdir / "test"
+    filename = tmpdir / f"test{DRIVER_EXT[driver]}"
```

Comment:
This was added because GDAL raises errors for some drivers if the extension is not what is required; i.e., even though we don't test OpenFileGDB in this block, it must have a `.gdb` extension.  This is an attempt to future-proof adding additional drivers to `DRIVERS`.